libexec/klab-prove-all

```diff
@@ -31,7 +31,7 @@ PNAME=$(jq -r '.name' < config.json)
 if [ "$PNAME" == "null" ]; then PNAME=""; fi
 export PNAME
 
-if [ -d ".git" ]; then
+if [ -d ".git" ] || [ -f ".git" ]; then
```

Comment:
Can you add a comment for future archaeologists to explain when `.git` would be a file?

Reply:
kthx :heart:

Reply:
Actually forget that, the info will be in the commit!